add Taranjot's ccid and username to team doc file

Taranjot-Singh opened the pull request:
Hello Team,

Adding my CCID and GitHub username for the project part 1 requirements. Can someone please review and approve this :) ?

cyrusdiego reviewed (changes requested):
Nit: looks like you may have tabbed in between your CCID and your GitHub user name, just for consistency do you mind just separating them by a single space.

Taranjot-Singh commented:
Hello Cyrus,

Thank you for catching that. I'm not sure what happened: in my local files, it shows just a space but here it shows a tab. So I removed the space, added it again, and made another commit. Now it seems to be fixed.
